.tsx files not resolving by default with webpacker 6.0.0-beta.2

I just upgraded to webpacker 6.0, following the steps in the "TypeScript" section of the [Integrations](https://github.com/rails/webpacker#integrations) documentation and I ran into an issue where webpacker could not process .tsx files by default:

```
ERROR in ./app/javascript/packs/application.tsx 6:0-45
Module not found: Error: Can't resolve '../src/Incremental' in '/Users/Mike/projects/frontendeval/app/javascript/packs'
```

My import line in `application.tsx`
```
import Incremental from '../src/Incremental';
```

Renaming this file to `Incremental.ts` worked, as well as adding the extension to the import line. I debugged a bit to see that the `webpackConfig` that ships in the `@rails/webpacker` package has this `resolve` configuration:
```
  resolve: {
    extensions: [ '.js', '.mjs', '.ts', '.coffee' ],
    ...
  },
```

So I ended up modifying `config/webpack/base.js` to be similar to the suggestion in the [CSS](https://github.com/rails/webpacker#css) suggestion in the documentation:
```
const { webpackConfig, merge } = require('@rails/webpacker');

const customConfig = {
  resolve: {
    extensions: ['.tsx'],
  },
};

module.exports = merge(webpackConfig, customConfig);
```
and everything works swimmingly.

From the documentation though, it really doesn't seem like this should be necessary. I noticed that the `extensions` configuration is gone from `webpacker.yml` so that's no longer where I'd go to customize this.

I know using `.ts` files for everything including TS files with JSX is viable alternative, but I like having the separation of having my files with JSX labeled by the extension. Am I doing something wrong? Is this behavior expected?

Dependencies section of my package.json
```
  "dependencies": {
    "@babel/core": "^7.0.0-0",
    "@babel/preset-react": "^7.10.4",
    "@babel/preset-typescript": "^7.12.7",
    "@rails/actioncable": "^6.0.0",
    "@rails/activestorage": "^6.0.0",
    "@rails/ujs": "^6.0.0",
    "@rails/webpacker": "^6.0.0-beta.2",
    "@types/react": "^16.9.44",
    "@types/react-dom": "^16.9.8",
    "autoprefixer": "^10.0.2",
    "axios": "^0.20.0",
    "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
    "postcss": "^8.0.9",
    "prop-types": "^15.7.2",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "tailwindcss": "^2.0.2",
    "ts-loader": "^8.0.2",
    "turbolinks": "^5.2.0",
    "typescript": "^4.1.3",
    "webpack": "^5.11.0"
  },
  "version": "0.1.0",
  "devDependencies": {
    "@typescript-eslint/eslint-plugin": "^3.7.1",
    "@typescript-eslint/parser": "^3.7.1",
    "@webpack-cli/serve": "^1.2.1",
    "eslint": "^7.6.0",
    "eslint-config-airbnb": "^18.2.0",
    "eslint-config-prettier": "^6.11.0",
    "eslint-plugin-import": "^2.22.0",
    "eslint-plugin-jsx-a11y": "^6.3.1",
    "eslint-plugin-prettier": "^3.1.4",
    "eslint-plugin-react": "^7.20.5",
    "eslint-plugin-react-hooks": "^4.0.8",
    "prettier": "^2.0.5",
    "webpack-cli": "^4.2.0",
    "webpack-dev-server": "^3.11.1"
  },
```

--- a/package/environments/base.js
+++ b/package/environments/base.js
@@ -91,7 +91,7 @@ module.exports = {
   },
   entry: getEntryObject(),
   resolve: {
-    extensions: ['.js', '.mjs', '.ts', '.coffee'],
+    extensions: ['.js', '.jsx', '.mjs', '.ts', '.tsx', '.coffee'],
     modules: getModulePaths(),
     plugins: [PnpWebpackPlugin]
   },

